src/main/java/views/monthly_menu/MonthMenuV.java

```diff
@@ -0,0 +1,188 @@
+            leftLayout.setBounds(30,30,200,200);
+            JPanel rightLayout = new JPanel(new BorderLayout(20, 20));
+            rightLayout.setBounds(30,30,333,200);
+
+            //Left side components: monthID
+            JPanel monthPanel = getMonthPanel(monthID);
+            leftLayout.add(monthPanel);
+
+            //Left side components: add/edit buttons
+            JPanel buttons = new JPanel(new GridLayout(0,2,20,20));
+            buttons.setBorder(BorderFactory.createEmptyBorder(10, 10, 10, 10));
+            addExpense.setBounds(200,50,200,30);
+            editExpense.setBounds(200,50,200,30);
+            addCategory.setBounds(200,50,200,50);
+            editCategory.setBounds(200,50,200,50);
+            buttons.add(addExpense);
+            buttons.add(editExpense);
+            buttons.add(addCategory);
+            buttons.add(editCategory);
+            leftLayout.add(buttons);
+
+            //Left side components: generate summary button
+            JPanel genSumButton = new JPanel();
+            genSumButton.setLayout(new BoxLayout(genSumButton, BoxLayout.LINE_AXIS));
+            genSumButton.setBorder(BorderFactory.createEmptyBorder(10, 10, 10, 10));
+            generateSummary.setBounds(200,50,200,50);
+            genSumButton.add(generateSummary);
+            leftLayout.add(genSumButton);
+
+            //Right side components: set JTables
+            Object[] expenseData = controller.getOutput(session, monthID).getExpenseData().toArray();
+            Object[] categoryData = controller.getOutput(session, monthID).getCategoryData().toArray();
+            JTable expenseTable = getExpenseTable((Expense[]) expenseData);
+            JTable categoryTable = getCategoryTable((Category[])categoryData);
+            Dimension dimension = new Dimension(333,180);
+            expenseTable.setPreferredScrollableViewportSize(dimension);
+            categoryTable.setPreferredScrollableViewportSize(dimension);
+
+            //Right side components: set JScrollPanes
+            JScrollPane expenseScroll = new JScrollPane(expenseTable, JScrollPane.VERTICAL_SCROLLBAR_ALWAYS,
+                    JScrollPane.HORIZONTAL_SCROLLBAR_NEVER);
+            expenseScroll.setBounds(15, 15, 333, 90);
+            rightLayout.add(expenseScroll, BorderLayout.NORTH);
+            JScrollPane categoryScroll = new JScrollPane(categoryTable, JScrollPane.VERTICAL_SCROLLBAR_ALWAYS,
+                    JScrollPane.HORIZONTAL_SCROLLBAR_NEVER);
+            categoryScroll.setBounds(15, 15, 333, 90);
+            rightLayout.add(categoryScroll, BorderLayout.CENTER);
+            rightLayout.setBorder(BorderFactory.createEmptyBorder(10, 10, 10, 10));
+
+            //Add both side to overall layout and set up frame
+            layout.add(leftLayout, BorderLayout.WEST);
+            layout.add(rightLayout, BorderLayout.CENTER);
+            frame.setContentPane(layout);
+        }
+        catch (Exception e){
```

Comment:
I think writing the specific Exception that is expected type here could be better

Reply:
Actually, as you point this out, I found that there may be some problem with this try catch. I'll get this figured out ASAP. Thanks a lot, Ari.

Reply:
The problem is fixed! As I confirmed again, there wasn't any exception for me to catch as I already caught in UpdateViewUCI, so I changed the try catch to try else.